### PR TITLE
change(web): KeymanWeb bundler-script centralization 📜

### PR DESCRIPTION
**Pull request by jahorton**
This PR converts _all_ KMW instances of the `build_bundler.js` bundling paradigm introduced during 17.0 toward modularization (during the 🧩 chain for #8560) with the single, centralized script introduced in this PR's direct ancestor - #10265.  Said script now accepts new parameters sufficient to reproduce the tailored configurations needed for each Web module.

After all, as noted on many of the scripts...

https://github.com/keymanapp/keyman/blob/0b8ee4e6dbec2a6503f07d8a61f9a3135a4d8e16/web/src/app/browser/build-bundler.js#L1-L6

These _were_ originally intended to be short-term... and as it turns out, cleaning up this loose end will facilitate achieving the goals of #10257.  (Otherwise, we'd have _even more_ build-bundler.js proliferation / WETness.)

----

Remaining references to a "build-bundler." script:

image

Yeah, I'm not touching `kmc`'s script.

@keymanapp-test-bot skip

### CODE REVIEW

File: web/src/tools/testing/recorder/build.sh [hunk 1]
```diff
@@ -15,6 +15,8 @@ SUBPROJECT_NAME=tools/testing/recorder
 # This script runs from its own folder
 cd "$THIS_SCRIPT_PATH"
 
+BUNDLE_CMD="node $KEYMAN_ROOT/common/web/es-bundling/build/common-bundle.mjs"
```
Comment (ermshiperete) [inline web/src/tools/testing/recorder/build.sh:18]: `BUNDLE_CMD` is already defined in `common.inc.sh`, so we wouldn't have to define it here.

File: web/src/tools/testing/bulk_rendering/build.sh [hunk 2]
```diff
@@ -14,6 +14,8 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 # This script runs from its own folder
 cd "$THIS_SCRIPT_PATH"
 
+BUNDLE_CMD="node $KEYMAN_ROOT/common/web/es-bundling/build/common-bundle.mjs"
```
Comment (ermshiperete) [inline web/src/tools/testing/bulk_rendering/build.sh:17]: What's the reason for not including `web/common.inc.sh`? Then we wouldn't have to define `BUNDLE_CMD` here...

**Review by ermshiperete — APPROVED** [review]
Two suggestions, but otherwise LGTM

File: common/web/gesture-recognizer/src/tools/build.sh [hunk 3]
```diff
@@ -76,8 +78,10 @@ fi
 
 if builder_start_action build :test-module; then
   tsc -b "$THIS_SCRIPT_PATH/unit-test-resources/tsconfig.json"
-  cd unit-test-resources
-  node build-bundler.js
-  cd ..
+
+  $BUNDLE_CMD    "${KEYMAN_ROOT}/common/web/gesture-recognizer/build/tools/obj/index.js" \
+    --out        "${KEYMAN_ROOT}/common/web/gesture-recognizer/build/tools/lib/index.mjs" \
+    --format "esm"
```
Comment (darcywong00) [inline common/web/gesture-recognizer/src/tools/build.sh:84]: Just a nit - you used `--format esm` everywhere else

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.239-alpha](https://keyman.com/downloads/releases/alpha/17.0.239)

**Pull request merged**